packages/flutter/test/material/expansion_panel_test.dart

```diff
@@ -5,18 +5,29 @@
 import 'package:flutter/material.dart';
 import 'package:flutter_test/flutter_test.dart';
 
+
 class SimpleExpansionPanelListTestWidget extends StatefulWidget {
   const SimpleExpansionPanelListTestWidget({
     Key key,
     this.firstPanelKey,
     this.secondPanelKey,
     this.canTapOnHeader = false,
+    this.expandedHeaderPadding
   }) : super(key: key);
 
   final Key firstPanelKey;
   final Key secondPanelKey;
   final bool canTapOnHeader;
 
+  /// If null the below default is applied
+  final EdgeInsets expandedHeaderPadding;
+
+  /// Mirrors the default expanded header padding as its source constants are private
+  static EdgeInsets defaultExpandedHeaderPadding()
```

Comment:
I've never used this pattern before, @HansMuller, what do you think?

Reply:
I spoke with Hans offline, and he says it looks fine, we just don't use this pattern very often :)